### PR TITLE
Contributing instructions page

### PR DESCRIPTION
**Pull request by Eddayavuz**
**Issue**

#143

**Changes Proposed**

1. Created a CONTRIBUTING.md file with instructions for contributing, in vuepress formatting.
1. Added a section to the README.markdown file to link to the contributing instructions

**Testing Instructions**

Please test by clicking on the link "contributing instructions" at the end of readme.md

**Extra Information**

Should I delete the Author section?

**Share with Us!**

This is my first contribution! Super excited.

### CODE REVIEW

**Review by jlooper — APPROVED** [review]
nicely done, Eda!

**Review by jlooper — CHANGES_REQUESTED** [review]
I'm not sure what the 'contrubuting' file is for, I don't think it's needed. We just want contributing.md and its link in README.md

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Contributing Instructions"
+---
+<TagLinks />
```
Comment (jlooper) [inline contributing.md]: we don't need this line, it's specific to the Recipe repo

File: contrubuting.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Contrubuting Instructions"
```
Comment (jlooper) [inline contrubuting.md]: I think this file should be deleted

**Review by jlooper — CHANGES_REQUESTED** [review]
Some changes to make please

**Comment by Eddayavuz** [comment]
Thank you for reviewing @jlooper , I appreciate your time! I've made the necessary changes.

File: README.md [hunk 3]
```diff
@@ -33,4 +33,4 @@ Workshop attendees and mentors can register for events at [FrontEndFoxes.org](ht
 
 The goal of the program is to familiarize women and those who identify as such with concepts of programming for web and mobile in a supportive and inclusive environment. Experienced developers, however, are very welcome to join in our efforts to organize events, support our scholarship fund, mentor a skulk, and provide content. We are grateful for all kinds of help!
 
-**Want to contribute? Please make sure to check out our [Code of Conduct](workshops/CODE_OF_CONDUCT.md).**
\ No newline at end of file
+**Want to contribute? Please make sure to check out our [Code of Conduct](workshops/CODE_OF_CONDUCT.md) as well as our [Contrubuting instructions](contributing.md).**
```
Comment (jlooper) [inline README.md]: one little fix: spelling of Contributing

**Review by jlooper — CHANGES_REQUESTED** [review]
almost perfect!

File: README.md (hunk 3 above)
Comment (Eddayavuz) [reply README.md]: Thank you! I did the spelling fix. 👍

**Review by jlooper — APPROVED** [review]

**Pull request merged**